source/renderer/app/containers/staking/dialogs/redeem-itn-rewards/Step1ConfigurationContainer.tsx

```diff
@@ -59,23 +77,13 @@ class Step1ConfigurationContainer extends Component<Props> {
     const selectedWallet: Wallet | null | undefined = allWallets.find(
       (current: Wallet) => current && current.id === selectedWalletId
     );
-    const { amount, isRestoring } = selectedWallet || {};
-    let errorMessage = null;
-
-    if (selectedWallet && !this.onWalletAcceptable(amount)) {
```

Comment:
This is where the bug was. We were only assigning the error message that the wallet is restoring, when wallet didn't meet the criteria to be selected.